src/components/ButtonPanel.js

```diff
@@ -1,38 +1,39 @@
+/* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
 import Button from './Button';
 
-function ButtonPanel() {
+function ButtonPanel(props) {
   return (
     <div>
       <div>
-        <Button name="AC" />
-        <Button name="+/-" />
-        <Button name="%" />
-        <Button name="÷" />
+        <Button name="AC" {...props} />
+        <Button name="+/-" {...props} />
+        <Button name="%" {...props} />
+        <Button name="÷" {...props} />
```

Comment:
We have a lot of repeated code. We can place all button names in an array and use the map method to generate all buttons for each group. For example:
```js
const grp1 = ["AC", "+/-", "%", "+"]
...
//jsx
{ 
  grp1.map(btnName => <Button name={btnName} {...props}/>) 
}
```